client/packages/common/src/intl/locales/en/common.json

```diff
@@ -416,5 +416,7 @@
   "sync-status.pull-remote": "Pull Remote",
   "sync-status.push": "Push",
   "table.show-columns": "Show / hide columns",
-  "warning.caps-lock": "Warning: Caps lock is on"
+  "warning.caps-lock": "Warning: Caps lock is on",
+  "heading.logout-confirm": "Logout?",
+  "messages.logout-confirm": "Are you sure you want to logout?"
```

Comment:
Wording here feels a little clumsy to me, but I can't think immediately what would be better